project-reports/2024/2024-Q3-Hyperledger-Fabric.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Fabric
+parent: 2024
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# 2024 Q3 Hyperledger Fabric
+
+# Project Health
+
+Hyperledger Fabric is fairly mature and stable with a v2.5 long-term support (LTS) release.
+There is less churn and fewer commits than in years past, with continued focus on quality, maintenance, and support.
+New features get proposed, approved, and implemented based on a community RFC process.
+Mailing list activity is down a bit compared to prior years.
+PRs and mailing list questions are generally turned around quickly.
+There is significant Discord activity.
+
+# Questions/Issues for the TOC
+
+None.
+
+# Releases
+
+LTS release:
+- v2.5 is the current LTS release with patch releases at least quarterly.
+- Note that v2.2 has reached end of maintenance and all users are encouraged to update to v2.5 to continue receiving fixes and security updates.
+
+[Releases](https://github.com/hyperledger/fabric/releases) past quarter:
+- Fabric v2.5.8 - May 21, 2024
+- Fabric v2.5.9 - June 18, 2024
+- Fabric v3.0.0-beta - March 14, 2024
+
+# Overall Activity in the Past Quarter
+
+The project delivered patch releases for the Fabric v2.5 long-term support release. See [release notes](https://github.com/hyperledger/fabric/releases) for details.
+
+v3.0 development is nearing completion. The following enhancements were made this quarter:
```

Comment:
it has been nearing completion for a year :-) 

Is there any ETA on the GA?

Reply:
We're down to the final v3.0 item - the lifecycle v1 removal and related updates. This work has been making steady albeit slow progress. As long as the progress continues my opinion is that we should complete that work prior to delivering the official v3.0 release.